tests/unit/lms/services/mailchimp_test.py

```diff
@@ -22,7 +31,7 @@ class TestSend:
                 {
                     "total_annotations": 2,
                     "annotators": [sentinel.annotator_1, sentinel.annotator_2],
-                    "courses": [sentinel.course_1, sentinel.course_2],
+                    "courses": [CourseDict(), CourseDict()],
```

Comment:
The new `{% if %}` in the template means that rendering the template now crashes if `courses` isn't a list of dicts with (at least some of) the expected keys, so had to replace these sentinels with dicts.